x-pack/plugins/alerting/common/alert.ts

```diff
@@ -94,6 +94,7 @@ export interface Alert<Params extends AlertTypeParams = never> {
   mutedInstanceIds: string[];
   executionStatus: AlertExecutionStatus;
   monitoring?: RuleMonitoring;
+  snoozeEndTime?: string;
```

Comment:
I'm not sure I understand why this is a string and not a Date?

Reply:
Can't this be `: Date | -1` like you're doing in the route rewriter?

Reply:
This type corresponds to what gets inserted into the SavedObject, not what comes from the `_snooze` route, so it's only ever going to be a Date, never `-1`.

Reply:
Okay cool so it makes sense to use `: Date` here?